build.gradle

```diff
@@ -3,73 +3,38 @@ apply plugin: 'java'
 sourceCompatibility = '1.8'
 [compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
 
-import org.gradle.api.internal.file.copy.CopySpecInternal
-
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'org.owasp:dependency-check-gradle:6.0.3'
-    }
-}
-
-apply plugin: 'org.owasp.dependencycheck'
-
 project.ext {
-
-    CC_VERSION = '8.3.0'
-    MXBUILD_VERSION = '8.12.0.2541'
+    CC_VERSION = '8.4.0'
+    mendixVersion = '8.12.7'
 }
 
-def runtimeLibs = "$buildDir/runtime/bundles"
 def userLibDir = "$projectDir/src/CommunityCommons/userlib"
 
-configurations {
-    tar
-}
-
 repositories {
-    mavenCentral()
-    ivy {
-		url 'https://cdn.mendix.com/'
-		layout 'pattern', {
-			artifact '/[organisation]/[module]-[revision].[ext]'
-		}
-		metadataSources {
-			artifact()
-		}
+    maven {
+        url 'https://nexus.rnd.mendix.com/repository/repo1-proxy/'

```

Comment:
Community Commons is developed in the open with contributions from the developer community. If external developers can't run the build, I would say that it matters.

Reply:
We depend on our internal repo for the public-api anyway, so I see no harm done here. It is my personal goal to publish these artefacts more publicly, if that works out I'll refactor this build so that it doesn't only work in our pipeline but also externally. Till then I don't see what I can do :)

Reply:
@jds-mendix The current build.gradle uses a workaround involving downloading mxbuild from CDN in order to fetch the public API jars, that works for everybody. :)

Reply:
@reinouts this would be really pointless, the next thing we're gonna do is to include a Gradle plugin for publishing. That's gonna come from the same place.

Reply:
Perhaps it would be possible to differentiate between a 'public' Gradle build file that allows you to open the project via Gradle in your IDE, and a 'private' one that includes the publish plugin and nexus dependencies. Otherwise we're making life more difficult for potential contributors.

Reply:
I've added a comment